lib/ansible/plugins/connection/kubectl.py

```diff
@@ -23,47 +23,127 @@
 DOCUMENTATION = """
     author:
         - xuxinkun
+    
     connection: kubectl
-    short_description: Run tasks in kubernetes pods
+    
+    short_description: Execute tasks in pods running on Kubernetes. 
+    
     description:
-        - Run commands or put/fetch files to an existing kubernetes pod container.
-    version_added: "2.0"
+        - Use the kubectl exec command to run tasks in, or put/fetch files to, pods running on the Kubernetes
+          container platform.
+    
+    version_added: "2.5"
+    
+    requirements:
+      - kubectl (go binary)
+    
     options:
-      kube_namespace:
+      kubectl_container:
         description:
-            - The namespace of the pod
+          - Container name. Required when a pod contains more than one container.
         default: ''
         vars:
-            - name: ansible_kube_namespace
-            - name: ansible_kubectl_namespace
+          - name: ansible_kubectl_container
```

Comment:
Should we also provide a env option for `kubectl_container`

Reply:
Doesn't seem like it would be necessary, as container names aren't really known ahead of time. I'm envisioning pod and container names coming from an inventory script.

Reply:
If pod has only one container, container name can be ignored. But if pod has more than one container, container name is necessary to be specified.

Reply:
Added.